mods/voidmulti.sc2mod/base.sc2data/gamedata/actordata.xml

```diff
@@ -74,7 +92,7 @@
         <On Terms="SelectionLocalUpdate.Nexus.Stop" Send="Destroy"/>
         <Icon value="Assets\Textures\RadarIcon2.dds"/>
         <IconArcLength value="1.750000"/>
-        <Range value="9.800000"/>
+        <Range value="10.000000"/>
```

Comment:
**undocumented**: Battery Overcharge's range indicator's range was raised from 9 to 10.

Most likely a visual bug fix.

Btw, why is there no range indicator when you manually target the recharge?